lib/cWarnings.ml

```diff
@@ -108,12 +108,12 @@ let () = CErrors.register_handler (function
     | WarnError w -> Some (print w)
     | _ -> None)
 
-let warn { Msg.tag; warning } ?loc v =
+let warn { Msg.tag; warning } ?loc ?quickfix v =
   let tag = DMap.tag_of_onetag tag in
   match warning_status warning with
   | Disabled -> ()
   | AsError -> Loc.raise ?loc (WarnError (W (tag, v)))
```

Comment:
Is it intended that AsError warnings don't have the quickfix?

Reply:
Indeed they can

Reply:
I'm swamped this week.  I guess the fix is about putting the quickfix argument also in `W`, @rtetley if you have a minute, help is welcome.

Reply:
I'll look into it, but most likely not before Wednesday